src/librbd/api/Trash.cc

```diff
@@ -116,27 +118,18 @@ int enable_mirroring(IoCtx &io_ctx, const std::string &image_id) {
 } // anonymous namespace
 
 template <typename I>
-int Trash<I>::move(librados::IoCtx &io_ctx, rbd_trash_image_source_t source,
-                   const std::string &image_name, uint64_t delay) {
+int Trash<I>::move_by_id(librados::IoCtx &io_ctx, rbd_trash_image_source_t source,
+                   const std::string &image_id, const std::string &image_name,
+                   uint64_t delay) {
   CephContext *cct((CephContext *)io_ctx.cct());
-  ldout(cct, 20) << "trash_move " << &io_ctx << " " << image_name
+  ldout(cct, 20) << "trash_move_by_id " << &io_ctx << " " << image_id
```

Comment:
nit: the debug prefix already includes the function name, so does not need adding "trash_move_by_id " here.

Reply:
Done.